assets/queries/k8s/cpu_limits/query.rego

```diff
@@ -0,0 +1,33 @@
+package Cx
```

Comment:
Check this implementation to a more specifiy searchKey

Attached rego query
[Uploading query.zip…]()